reana_db/models.py

```diff
@@ -470,6 +470,50 @@ def __repr__(self):
         return "<InteractiveSession %r>" % self.name
 
 
+class WorkflowService(Base):
+    """Workflow Service table."""
+
+    __tablename__ = "workflow_service"
+    __table_args__ = {"schema": "__reana"}
+
+    workflow_id = Column(UUIDType, ForeignKey("__reana.workflow.id_"), nullable=True)
+    service_id = Column(UUIDType, ForeignKey("__reana.service.id_"), primary_key=True)
+
+    def __repr__(self):
+        """Workflow Service string representation."""
+        return f"<WorkflowService {self.service_id} {self.workflow_id}>"
+
+
+class ServiceType(enum.Enum):
+    """Enumeration of service types."""
+
+    dask = 0
+
+
+class Service(Base):
+    """Service table."""
+
+    __tablename__ = "service"
+    id_ = Column(UUIDType, primary_key=True, default=generate_uuid)
+    name = Column(String(255))
+    uri = Column(Text)  # uri to access the service
+    status = Column(Enum(RunStatus), nullable=False, default=RunStatus.created)
```

Comment:
For admin use cases, it may be perhaps good to have some indexes, e.g. on the `status` column so that we could ask "which users have Dask dashboards still open?".  But this is something that we may also get to later.